src/Symfony/Component/Asset/UrlPackage.php

```diff
@@ -129,7 +129,7 @@ private function getSslUrls($urls)
         foreach ($urls as $url) {
             if ('https://' === substr($url, 0, 8) || '//' === substr($url, 0, 2)) {
                 $sslUrls[] = $url;
-            } elseif ('http://' !== substr($url, 0, 7)) {
+            } elseif (false === strpos($url, '://')) {
```

Comment:
If I have `://` in my query string, this will now return true, can you add a testcase for that?

Reply:
@iltar fixed